### DISCUSSION TITLE
Removed models in entrypoint.sh

### DISCUSSION THREAD

**Discussion by shreya-dama-Crafty-Apes-VFX** [AbdBarho/stable-diffusion-webui-docker#509] [Q&A]
Some of the models such as hed,mlsd,leres, uniformer are missing on the new update made to the entrypoint.sh . I wanted to know if there is any particular reason for that

**Comment by AbdBarho** [comment]
This should be fixed now with https://github.com/AbdBarho/stable-diffusion-webui-docker/releases/tag/8.0.0
